### PR TITLE
page: Result Page with functionality ft. share quiz

### PR DESCRIPTION
**Pull request by manojsarna**
- [x] Result Page Added
- [x] Functionality Added : Share Quiz

image

image

### CODE REVIEW

File: src/components/quiz-card/QuizCard.jsx [hunk 1]
```diff
@@ -1,13 +1,31 @@
 import { Link } from "react-router-dom";
+import { toast } from "react-toastify";
 import "./quizcard.css";
 export function QuizCard({ quiz }) {
   return (
-    <Link to={`/quiz/${quiz._id}`} className="sm-quiz-link">
+    <Link
+      to={`/quiz/${quiz._id}`}
+      className="sm-quiz-link"
+      title={`Go To ${quiz.title} Quiz`}
```
Comment (iRohitGaur) [inline src/components/quiz-card/QuizCard.jsx:9]: appreciation: nice use of title

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: src/pages/result/Result.jsx [hunk 2]
```diff
@@ -1,4 +1,60 @@
+import { useLocation, useNavigate } from "react-router-dom";
 import "./result.css";
 export function Result() {
-  return <></>;
+  const navigate = useNavigate();
+  const { state } = useLocation();
+
+  console.log(state);
```
Comment (pankaj-prog) [inline src/pages/result/Result.jsx]: remove console

**Review by pankaj-prog — APPROVED** [review]
Ui is nice,
 You can also ask for confirmation on quitting the quiz.

**Pull request merged**